profiles/twelvestepintergroup/themes/twelvesteptheme/css/style.css

```diff
@@ -1,3 +1,61 @@
 /**
  * Place your custom styles here.
  */
+
+div.view-meetings #view-field-time-table-column {
+  width: 20%;
+}
+div.view-meetings #view-field-format-table-column {
+  width: 30%;
+}
+
+/**
+ * Make the meeting table responsive.
+ *
+ * Copied from https://css-tricks.com/examples/ResponsiveTables/responsive.php.
+ *
+ * @todo: have a themer review this. Something in tables.less is interfering
+ * around 600-760px.
+ */
+@media only screen and (max-width: 600px) {
+
+  /* Force table to not be like tables anymore */
+  div.view-meetings table,
```

Comment:
Without using less, can this be done without repeating div.view-meetings all over the place?